remoteworkitem/trackerquery_repository.go

```diff
@@ -117,3 +119,57 @@ func (r *GormTrackerQueryRepository) Save(ctx context.Context, tq app.TrackerQue
 
 	return &t2, nil
 }
+
+// Delete deletes the tracker query with the given id
+// returns NotFoundError or InternalError
+func (r *GormTrackerQueryRepository) Delete(ctx context.Context, ID string) error {
+	var tq = TrackerQuery{}
+	id, err := strconv.ParseUint(ID, 10, 64)
+	if err != nil {
+		// treat as not found: clients don't know it must be a number
+		return NotFoundError{entity: "trackerquery", ID: ID}
+	}
+	tq.ID = id
+	tx := r.ts.TX()
+	tx = tx.Delete(tq)
+	if err = tx.Error; err != nil {
+		return InternalError{simpleError{err.Error()}}
+	}
+	if tx.RowsAffected == 0 {
+		return NotFoundError{entity: "trackerquery", ID: ID}
+	}
+	return nil
+}
+
+// List returns tracker query selected by the given criteria.Expression, starting with start (zero-based) and returning at most limit items
+func (r *GormTrackerQueryRepository) List(ctx context.Context, criteria criteria.Expression, start *int, limit *int) ([]*app.TrackerQuery, error) {
+	where, parameters, err := models.Compile(criteria)
```

Comment:
models.Compile(...) will not really work for Tracker items. Please remove this support or generalize the tracker.